projects/project-player-library/src/lib/pages/task-details-page/task-details-page.component.ts

```diff
@@ -154,10 +156,25 @@ export class TaskDetailsPageComponent implements OnInit {
         if(data.isChecked && data.upload){
           this.routingService.navigate(`/add-files/${this.projectDetails._id}`,{taskId:this.taskId})
         }else{
-          this.toasterService.showToast('ACCEPT_POLICY_ERROR_MSG')
+          this.toasterService.showToast('ACCEPT_POLICY_ERROR_MSG',"danger")
         }
       }
     })
   }
 
+  onDateChange(newDate: Date) {
+    let localDateString = this.formatDateToLocal(newDate);
+    this.task.endDate = localDateString;
+    this.updateDataInDb();
+  }
+
+  formatDateToLocal(date: Date): string {
```

Comment:
No need of this function

Reply:
this function returns the latest date because the date directly was getting declared previous one